### DISCUSSION TITLE
$spacer multiplier

### DISCUSSION THREAD

**Discussion by Bot12313** [twbs/bootstrap#32408] [Ideas] [answered]
While overviewing Bootstrap SCSS (v4.5.3) sources, I've noticed one confusing thing about $spacer variable.
Here it's description from comments:
```
// Spacing
//
// Control the default styling of most Bootstrap elements by modifying these
// variables. Mostly focused on spacing.
// You can add more entries to the $spacers map, should you need more variation.

```
Wasn't it more convenient and intuitive to multiply every padding/margin value in "variables" (I often do it for my own themes)?

---

Moreover I've found $spacer usage outside the **_variables.scss** file.

**_images.scss**
```
.figure-img {
  margin-bottom: $spacer / 2;
  line-height: 1;
}
```
  
**_tables.scss**
```
.table {
  width: 100%;
  margin-bottom: $spacer;
  color: $table-color;
  background-color: $table-bg; // Reset for nesting within parents with `background-color`.
...
```
  
**_type.scss**
```
// Blockquotes
.blockquote {
  margin-bottom: $spacer;
  @include font-size($blockquote-font-size);
}
```
  
Should them have their own variables inside **_variables.scss** just like the other values do?

**Comment by mdo** [comment, marked as answer]
`$spacer` gets reused and extended into a Sass map. We use `$spacer` as a general baseline that we can scale things around.

https://github.com/twbs/bootstrap/blob/8fed098ef9a60994df2389efa194b9af2b6cf520/scss/_variables.scss#L239-L247
